can/interfaces/hookspecs.py

```diff
@@ -0,0 +1,11 @@
+from pluggy import HookspecMarker
+
+
+hookspec = HookspecMarker("pythoncan")
+
+
+@hookspec
+def pythoncan_interface(interface):
+    """ return tuple (module, classname) containing python-can interface info
```

Comment:
What does it return if the plugin doesn't support the given interface? None/False/Exception?

This would return the `modulename` not the `module`.
Please document the parameter interface (`str`).